public/reindex_search.sh

```diff
@@ -0,0 +1,457 @@
+#!/bin/bash
+
+# reindexes elasticsearch
+
+# loads elastic suffix from .env file
+ELASTIC_SUFFIX=$(grep ELASTIC_SUFFIX ../.env | xargs)
+IFS='=' read -ra ELASTIC_SUFFIX <<< "$ELASTIC_SUFFIX"
+ELASTIC_SUFFIX=${ELASTIC_SUFFIX[1]//$'\r'}
+
+if [[ $ELASTIC_SUFFIX == "local" ]]; then
+  ELASTIC_ENDPOINT="localhost:9200"
+fi
+
+if [[ $ELASTIC_SUFFIX == "dev" ]]; then
+  ELASTIC_ENDPOINT="https://vpc-elasticsearch-ccs-dev-wvwrvhqdfoxbuhcdft7bk33dta.eu-west-1.es.amazonaws.com"
+fi
+
+if [[ $ELASTIC_SUFFIX == "uat" ]]; then
+  ELASTIC_ENDPOINT="https://vpc-elasticsearch-ccs-uat-ebfgjzsrb2rdyxm6z3oy2mnohi.eu-west-2.es.amazonaws.com"
+fi
+
+if [[ $ELASTIC_SUFFIX == "prod" ]]; then
+  ELASTIC_ENDPOINT="https://vpc-elasticsearch-ccs-prod-eur4vilhtjrxeumhbeqevtkcvi.eu-west-2.es.amazonaws.com"
```

Comment:
Those value cannot be in code base, they will have to be store inside the env file for different environment.

Reply:
ah ok thanks. Yep that makes sense.